extensions/amp-facebook-comments/0.1/amp-facebook-comments.js

```diff
@@ -80,6 +80,7 @@ class AmpFacebookComments extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    this.element.setAttribute('data-embed-as', 'comments');
```

Comment:
This is safe to set directly because while `data-*` attributes are valid (and therefore there could be a pre-existing `data-embed-as` value that would be overwritten by this operation), it would have been noop and not properly propagated.

Reply:
We also do want to overwrite any existing value for `data-embed-as`, otherwise all `amp-facebook-*` components could embed post types from others.